Code/Tools/SceneAPI/SceneCore/Mocks/DataTypes/ManifestBase/MockISceneNodeSelectionList.h

```diff
@@ -26,27 +26,24 @@ namespace AZ
                 // IMeshGroup
                 MOCK_CONST_METHOD0(GetSelectedNodeCount,
                     size_t());
-                MOCK_CONST_METHOD1(GetSelectedNode,
-                    const AZStd::string & (size_t index));
                 MOCK_METHOD1(AddSelectedNode,
-                    size_t(const AZStd::string & name));
-                size_t AddSelectedNode(AZStd::string&& name)
+                    void(const AZStd::string & name));
+                void AddSelectedNode(AZStd::string&& name)
                 {
-                    return AddSelectedNode(name);
+                    AddSelectedNode(name);
```

Comment:
The name should actually be moved here.
```suggestion
                    AddSelectedNode(AZStd::move(name));
```